mempool/clist_mempool_test.go

```diff
@@ -458,9 +450,9 @@ func TestMempoolMaxMsgSize(t *testing.T) {
 
 		tx := tmrand.Bytes(testCase.len)
 		err := mempl.CheckTx(tx, nil, TxInfo{})
-		msg := &TxMessage{tx}
-		encoded := cdc.MustMarshalBinaryBare(msg)
-		require.Equal(t, len(encoded), txMessageSize(tx), caseString)
+		bz := gogotypes.BytesValue{Value: tx}
```

Comment:
Nit on consistency: `bz` usually is used for the resulting bytes for encoding (e.g. what is called `encoded` below).